vexination/walker/Cargo.toml

```diff
@@ -18,8 +18,8 @@ hex = "0.4.3"
 packageurl = "0.3"
 rand = "0.8"
 csaf = "0.5.0"
-csaf-walker = { version = "=0.6.0-alpha.6", default-features = false, features = ["crypto-openssl"] }
-walker-common = { version = "=0.6.0-alpha.6", features = [] }
+csaf-walker = { path = "../../../csaf-walker/csaf", default-features = false, features = ["crypto-openssl", "csaf"] }
```

Comment:
We cannot merge this. For testing of a local version, you can use the `[patch]` section in the root level `Cargo.toml` file: https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html